tests/functional/l/logical_tautology.py

```diff
@@ -11,21 +11,21 @@ def foo():
         return True
     elif arg <= arg: # [comparison-with-itself]
         return True
-    elif None == None: # [comparison-with-itself]
+    elif None == None: # [comparison-of-constants, comparison-with-itself]
```

Comment:
As we're updating these comments as well, could you add the double spaces here?

Reply:
I'll try using the autoformatter locally on my machine.

Reply:
Did this work?